pull_request_template.md

```diff
@@ -1,25 +1,36 @@
 <!-- First of all, THANK YOU for your contribution. -->
 
 ## Changes Proposed:
--  
--  
+-  [ ] This pull request promotes changes to the core (units, players, creatures, game systems).
+-  [ ] This pull request promotes changes to scripts (bosses, spell scripts, creature scripts).
+-  [ ] This pull request promotes database changes.
+
+If your pull request promotes complex changes that require a detailed explanation, please describe them in detail specifying what your solution is and what is it meant to address.
 
 ## Issues Addressed:
 <!-- If your fix has a relating issue, link it below -->
 - Closes 
 
 ## SOURCE:
 <!-- If you can, include a source that can strengthen your claim -->
+- [ ] The changes promoted by this pull request have been validated through live research (checked on live servers)
+- [ ] The changes promoted by this pull request have been validated through sniffs (remember to share them with the opensource community!)
+- [ ] The changes promoted by this pull request come from a partially or entirelly from another project (cherry-pick). **Cherry-picks must be commited using the proper --author tag in order to be accepted, thus crediting the original authors, unless otherwise unable to be found**
 
 ## Tests Performed:
 <!-- Does it build without errors? Did you test in-game? What did you test? On which OS did you test? Describe any other tests performed -->
-- 
-- 
+- [ ] This pull request has been tested in-game by the author.
+- [ ] This pull request has been tested by the community.
+- [ ] This pull request has requires further testing and may have edge cases to be tested.
```

Comment:
L#24 & L#32 basically the same

Reply:
Wanted an option for sensitive prs/prs that may break stuff and may require careful testing and following the reproduction steps in the issue may not be enough, but I probably worded poorly

Reply:
I get that, but I think you can remove it form L#32 and just keep it on L#24

Reply:
> Wanted an option for sensitive prs/prs that may break stuff and may require careful testing and following the reproduction steps in the issue may not be enough, but I probably worded poorly

I know that this does happen with some PRs, but I guarantee to you not every contributor that fills out this template will know or even have an idea if more thorough testing will be required.